projects/packages/my-jetpack/_inc/components/my-jetpack-screen/style.scss

```diff
@@ -0,0 +1 @@
+@import '~@automattic/jetpack-base-styles/style';
```

Comment:
Just curious about why we need to put in all places this impor, it's not injected globally?

Reply:
I don't know, are they?

Here again, I'm hoping we refactor all this CSS very soon.

By very soon I mean before someone starts to use and customize these elements using their classes

Reply:
It's my understanding that we're moving away from importing the [`style.scss`](https://github.com/Automattic/jetpack/blob/master/projects/js-packages/base-styles/style.scss) anyway, as those styles get applied globally and affect non-RNA parts of the page.

So I think it's ok to keep importing this styles file as a temporary workaround as is until we get the styling roadblock resolved.

Reply:
Agreed @sergeymitr